pkgs/applications/networking/instant-messengers/pidgin-plugins/skype4pidgin/default.nix

```diff
@@ -2,10 +2,11 @@
 , dbus-glib }:
 
 stdenv.mkDerivation rec {
-  name = "skype4pidgin-novas0x2a-20120411-6c53f7c48f";
+  name = "skype4pidgin-novas0x2a";
```

Comment:
This package should just be dropped.
It hasn't worked for years, since the old skype for linux was retired.